crates/common/network_spec/src/networks/lean.rs

```diff
@@ -138,3 +141,6 @@ impl<'de> Deserialize<'de> for DiscardUnknown {
         Ok(DiscardUnknown)
     }
 }
+
+pub static TEST: LazyLock<Arc<LeanNetworkSpec>> =
+    LazyLock::new(|| LeanNetworkSpec::ephemery().into());
```

Comment:
Since this is used nowhere else, y ou could honestly just inline it in the function